src/client/game/game-reducer.js

```diff
@@ -47,104 +55,145 @@ const game = (state = initialState, action) => {
                 disableConsonant: action.payload.disableConsonant,
                 disableVowel: action.payload.disableVowel
             });
-        case actionTypes.GET_CONSONANT:
+        case actionTypes.DISABLE_START:
             return updateState(state, {
-                letterList: [...state.letterList, action.payload]
+                disableStart: action.payload
+            });
+        case actionTypes.ROUND_STARTED:
+            return updateState(state, {
+                roundResults: false
+            });
+        case actionTypes.ROUND_ENDED:
+            return updateState(state, {
+                roundResults: true
+            });
+        case actionTypes.RESET_ROUND:
+            return updateState(state, {
+                letterList: [],
+                answerTimerValue: null,
+                submissionTimerValue: null,
+                disableStart: true,
+                answering: false,
+                submission: false,
+                roundResults: false,
+                finalAnswers: [],
+                resetRound: true,
+                randomNumber: null,
+                numberList: []
+            });
+        case actionTypes.RESET_FINISHED: {
+            return updateState(state, {
+                resetRound: false,
+                disableConsonant: false,
+                disableVowel: false,
+                disableLarge: false,
+                disableSmall: false
+            });
+        }
+        case actionTypes.GAME_PARAMETERS:{
+            return updateState(state,{
+                gameParams: action.payload
             });
-        case actionTypes.GET_VOWEL:
+        }
+        case actionTypes.GAME_FINISHED: {
+            return updateState(state, {
+                gameFinished: true
+            });
+        }
+        case actionTypes.NEXT_ROUND_TYPE: {
+            return updateState(state, {
+                nextRoundType: action.payload
+            });
+        }
+
+        //letters round actions
```

Comment:
Don't think this comment is needed. By splitting out the actions we could have a better reducer/store as a result. Let's do a session on this later on in the week